### ISSUE TITLE
Error in example of FTS method

### ISSUE THREAD

**Issue by Samlesh** [SSAGESproject/SSAGES#35]
Dear Team,
I was executing the examples section in the FTS folder of examples on Gromacs 5.1.3. I could get the desired results of examples having  ADP in vacuum for both a single processor and two processors per walker. However, when I tried to run the ADP water environment with the provided adp_H2O.tpr file, Gromacs gave an error of could not settle bad contacts with water. So I did an energy minimization step externally on the provided gro file initially and followed the same procedure in README.md to execute. However, the same bad contact error popped up after running 8 ps. What I think is when the new image position is generated, I am doubtful if the software is having an energy minimization inbuilt to remove bad contacts. Can you please help me out to run this example of  ADP in water in Gromacs?

<img width="683" alt="Screenshot 2021-06-26 at 7 43 56 PM" src="https://user-images.githubusercontent.com/82301782/123515863-e2a83b00-d6b6-11eb-9365-f571c5e19a1a.png">

**Comment by pabloferz** [comment]
Hi @Samlesh, how are you running the energy minimization step? Have you tried running it longer?

**Comment by Samlesh** [comment]
Hi @pabloferz, like previous examples of FTS, in this ADP+water example, initially I tried to directly run the tpr file using SSAGES( without any change from my side) as per the procedure in the Readme file and I got the above bad water contact error. Then I took the gro file from this ADP+WATER folder and energy minimized it externally and generated a new tpr file using the energy minimized gro file and mdp file given in this folder. And again when I submitted the job same error came after running for 8 ps. As far as results of energy minimization is concerned it was carried out for 10 ps. But since after energy minimisation , the SSAGES software ran till 8 ps , i guess there might be  some inbuilt issue which is causing this problem whenever we have more than one component?(like ADP and water)
